src/pages/[section]/[...id].astro

```diff
@@ -28,4 +49,16 @@ const { Content } = await render(entry)
     )
   }
   <Content />
+  {
+    propsData
+      .filter((comp: any) => !!comp)
+      .map((component: any) => (
```

Comment:
Would be something in a future update, but wondering if we want to tweak the order the prop tables get rendered. Believe currently it's just based on whatever order the propComponents array in frontmatter is written, but would sorting alphabetically make more sense? Or some other way to dictate what the order should be rather than manually having to know "Component X should be rendered before Component C because Reasons"

Reply:
I _think_ I lean towards just letting the order be dictated by the `propsData` order? Just because it's the simplest solution and lets the order be dictated consumer side. But I'm not firmly against us sorting it some way theoretically.